BaseFilter/sections/specific.txt

```diff
@@ -582,6 +582,7 @@ $subdocument,third-party,denyallow=disqus.com,domain=welovemanga.one
 ! NOTE: Specific rules
 !
 !
+||4everproxy.com/direct/aHR0cHM6Ly9rbG9uZW5vcm0uY29tL2
```

Comment:
This is an anonimizer. Any site can be loaded through 4everproxy.com and we can duplicate all rules for it)